panels/templates/panel_app_form.html

```diff
@@ -0,0 +1,136 @@
+<script type="text/javascript">
+    var showOrHideOtherPresentation = function () {
+        setVisible($.field('other_presentation').parents('.form-group'), $.val('presentation') === {{ c.OTHER }});
+    };
+    var showOrHideOtherLength = function() {
+        setVisible($('#other-length'), $.val('length') === {{ c.OTHER }});
+        setVisible($.field('length_reason').parents('.form-group'), $.val('length') != {{ c.SIXTY_MIN }});
+    };
+    var showOrHideTablesDesc = function() {
+        setVisible($('#tables-desc'), $.field('need_tables').prop('checked'));
+    };
+    var showOrHideCostDesc = function() {
+        setVisible($('#cost-desc'), $.field('has_cost').prop('checked'));
+    };
+
+    $(function () {
+        showOrHideOtherPresentation();
+        showOrHideOtherLength();
+        showOrHideTablesDesc();
+        showOrHideCostDesc();
+        $.field('presentation').on('change', showOrHideOtherPresentation);
+        $.field('length').on('change', showOrHideOtherLength);
+        $.field('need_tables').on('click', showOrHideTablesDesc);
+        $.field('has_cost').on('click', showOrHideCostDesc);
+    });
+</script>
+<div class="form-group">
+    <label class="col-sm-3 control-label">Panel Name:</label>
+    <div class="col-sm-6">
+        <input class="form-control" type="text" name="name" value="{{ app.name }}" />
+    </div>
+</div>
+<div class="form-group">
+    <label class="col-sm-3 control-label">Type of Panel:</label>
+    <div class="col-sm-6">
+        <select name="presentation" class="form-control">
+            {{ options(c.PRESENTATION_OPTS, app.presentation) }}
+        </select>
+    </div>
+</div>
+<div class="form-group">
+    <label class="col-sm-3 control-label">Please Describe:</label>
+    <div class="col-sm-6">
+        <input type="input" class="form-control" name="other_presentation" value="{{ app.other_presentation }}" />
+    </div>
+</div>
+<div class="form-group" style="margin-bottom:0px">
+    <label class="col-sm-3 control-label">Panel Description</label>
+    <div class="col-sm-6">
+        <textarea class="form-control" name="description" rows="4">{{ app.description }}</textarea>
+    </div>
+</div>
+<div class="form-group">
+    <p class="help-block col-sm-9 col-sm-offset-3">
+        Will appear as a subheading on the schedule. 200 words max.
+    </p>
+</div>
+<div class="form-group" style="margin-bottom:0px">
+    <label class="col-sm-3 control-label">Expected Panel Length:</label>
```

Comment:
Most of this is copy-pasted from the panel form, since we were repeating the same code in two places. The new fields start here and end at Technical Needs.